### PR TITLE
Random housekeeping

### PR DESCRIPTION
**Pull request by johnjohndoe**
# Description
- Here is a few unrelated code improvements suggested by Lint and such.
- Please refer to the individual commit messages when review this pull request.

### CODE REVIEW

File: app/k9mail/src/main/java/com/fsck/k9/widget/unread/UnreadWidgetDataProvider.kt [hunk 1]
```diff
@@ -23,12 +23,10 @@ class UnreadWidgetDataProvider(
     private val folderNameFormatterFactory: FolderNameFormatterFactory
 ) {
     fun loadUnreadWidgetData(configuration: UnreadWidgetConfiguration): UnreadWidgetData? = with(configuration) {
-        if (SearchAccount.UNIFIED_INBOX == accountUuid) {
-            loadSearchAccountData(configuration)
-        } else if (folderId != null) {
-            loadFolderData(configuration)
-        } else {
-            loadAccountData(configuration)
+        when {
```
Comment (cketti) [inline app/k9mail/src/main/java/com/fsck/k9/widget/unread/UnreadWidgetDataProvider.kt:26]: Can you please revert this change. I find it makes the code harder to read than the old version.

Comment (johnjohndoe) [reply app/k9mail/src/main/java/com/fsck/k9/widget/unread/UnreadWidgetDataProvider.kt]: :heavy_check_mark: Changed in 117af496994763ef11c5265777cc4cef3dcdce43.
